Tests/TestingApps/PaywallsTester/PaywallsTester/Configuration.swift

```diff
@@ -33,7 +33,7 @@ final class Configuration: ObservableObject {
     #warning("Configure API key if you want to test paywalls from your dashboard")
     // Note: you can leave this empty to use the production server, or point to your own instance.
     private static let proxyURL = ""
-    private static let apiKey = ""
+    private static let apiKey = "appl_TFIZqBlauHeMhnqfNlWWFBjZwkf"
```

Comment:
❗

Reply:
How should I include this for the TF build?

Reply:
CircleCI env variable + variable replacement in a script, you can see how we do it for the backend integration tests app